integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXImageValidatorTest.java

```diff
@@ -0,0 +1,39 @@
+package com.sequenceiq.it.cloudbreak.testcase.e2e.distrox;
+
+import javax.inject.Inject;
+
+import org.testng.annotations.Test;
+
+import com.sequenceiq.it.cloudbreak.client.DistroXTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.testcase.e2e.ImageValidatorE2ETest;
+
+public class DistroXImageValidatorTest extends ImageValidatorE2ETest {
```

Comment:
I think this class should not extend `ImageValidatorE2ETest`, since `InternalSdxDistroxTests` also creates DistroX cluster